plugins/woocommerce/includes/wc-stock-functions.php

```diff
@@ -197,11 +197,22 @@ function wc_reduce_stock_levels( $order_id ) {
 		$item->add_meta_data( '_reduced_stock', $qty, true );
 		$item->save();
 
-		$changes[] = array(
+		$change = array(
```

Comment:
This `=` sign should be aligned with the one below as per WP coding standards.

```suggestion
		$change    = array(
```